addons/osm-edge/enable

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+read -ra ARGUMENTS <<< "$@"
+
+ARCH=$(arch)
+
+# check if osm-edge cli is already in the system.  Download if it doesn't exist.
+if [ ! -f "${SNAP_DATA}/bin/osm" ]; then
+  OSM_EDGE_VERSION="${OSM_EDGE_VERSION:-v1.1.0}"
+  echo "Fetching osm-edge version $OSM_EDGE_VERSION."
+
+  run_with_sudo mkdir -p "${SNAP_DATA}/tmp/osm-edge"
+  (cd "${SNAP_DATA}/tmp/osm-edge"
```

Comment:
We are missing the closing parenthesis.

Reply:
Thanks for the catch.  I've fixed missing parenthesis issue.